src/sentry/shared_integrations/client/proxy.py

```diff
@@ -51,6 +53,11 @@ def infer_org_integration(
     return org_integration_id
 
 
+def get_proxy_url() -> str:
+    control_address = getattr(settings, "SENTRY_CONTROL_ADDRESS", "")
```

Comment:
You should be able to use `settings.SENTRY_CONTROL_ADDRESS` now as it is always defined.